facter/facter.php

```diff
@@ -213,7 +213,11 @@ function Update()
 	if($resultarray) {
 		$id=$resultarray[0]['id'];
 		// Update SW type (OS)
-		$osrelease = $facter['operatingsystem'] . '%GSKIP%' . $facter['operatingsystem'] . ' V' . $facter['operatingsystemrelease'];
+		$os_release_lts = "";
+		if ($facter['operatingsystem'] == 'Ubuntu' && explode(".", $facter['operatingsystemrelease'])[0] % 2 == 0) {
```

Comment:
If you want, why not add the link. A sample string may be helpful in case someone decides to change this code later, then they will be able to test it.